graphtv/maxflow.cpp

```diff
@@ -500,6 +500,13 @@ Graph::flowtype Graph::maxflow()
 	}
 
 	delete nodeptr_block;
+
+	printf("\ntr_cap in this run is: \n");
+  for (i=node_block->ScanFirst(); i; i=node_block->ScanNext()) {
+    printf("%f \n",i->tr_cap);
+  }
+
+

```

Comment:
Does this `maxflow` function ever get called?  Dead code should be removed

Reply:
this maxflow is needed, as the parametric maxflow is calling the last two functions in this file.